server/IdentityService/Controllers/IdentityController.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using IdentityService.Dto;
+using IdentityService.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace IdentityService.Controllers
+{
+    [ApiController]
+    [Route("api/identity")]
+    public class IdentityController : Controller
+    {
+        private readonly IIdentityService _identityService;
+
+        public IdentityController(IIdentityService identityService)
+        {
+            _identityService = identityService;
+        }
+
+        /// <summary>
+        /// Checks if given credentials are valid and returns token
+        /// </summary>
+        /// <param name="login">Login of user</param>
+        /// <param name="password">Password of user</param>
+        /// <response code="200">Returns object with accessToken</response>
+        /// <response code="401">Returned if credentials are not valid</response>
+        [HttpPost]
+        [Route("login")]
+        public IActionResult Login([FromQuery] string login, [FromQuery] string password)
+        {
+            try
+            {
+                var token = _identityService.Login(login, password);
+                return new OkObjectResult(new LoginResponse(token));
+            }
+            catch (NullReferenceException e)
```

Comment:
Personally I think handling this by NRE could blur the situation if an exception not related to wrong credentials occurs. For example we could forget to register something in DI and quite possibly we'd get a NRE this way. Personally I'd go with something like 
``` bool TryLoginUser(login, password, out token) ```
Just like with int32.TryParse(...)